lib/http/configs.js

```diff
@@ -88,7 +88,12 @@ export function addRoutes({app, service}) {
       if(validateConfigFn) {
         const {valid, error} = await validateConfigFn({config, op: 'create'});
         if(!valid) {
-          throw error;
+          throw new BedrockError(
+            'ConfigFn validation failed.', 'ValidationError', {
```

Comment:
Fixing up the message here -- but also it can't be a `ValidationError` because it doesn't include JSON schema errors via the `errors` property in the details. So let's use a more general `DataError` instead:

```suggestion
            'Configuration validation failed.', 'DataError', {
```

Reply:
Updated [df17951](https://github.com/digitalbazaar/bedrock-service-core/pull/16/commits/df17951982cc712f572647aa28a7f55df6b91c42)